camelot/core/serializable.py

```diff
@@ -69,6 +74,16 @@ def default(self, obj):
             return obj.toVariant()
         if isinstance(obj, QtGui.QColor):
             return obj.name()
+        if isinstance(obj, (Decimal, datetime.date, datetime.datetime)):
+            return initial_naming_context._bind_object(obj)
+        if isinstance(obj, StoredFile):
```

Comment:
idem hier, die StoredFile zou nt tot hier mogen geraken, in de delegate zou de relevante info in de role moeten w gestoken